### PR TITLE
Make interface state delegate non optional

### PR DESCRIPTION
**Pull request by wsdwsd0829**
This is to remove respondToSelector check and have better performance.

### CODE REVIEW

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/48f2b73758bba7ae21a4453aff077af7/raw/43af9c7677aa718507145a5837545ffd4c8c7cc0/log.txt)

**Comment by maicki** [comment]
In theory this gonna be an API change. @nguyenhuy @Adlai-Holler @garrettmoon any thoughts?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## master
 * Add your own contributions to the next release on the line below this with your name.
+- [ASDisplayNode] Make interface state delegate protocol required. [Max Wang](https://github.com/wsdwsd0829). [#1112](https://github.com/TextureGroup/Texture/pull/1112)
```
Comment (nguyenhuy) [inline CHANGELOG.md]: Let's highlight that it's a breaking change.

**Comment by maicki** [comment]
@wsdwsd0829 Would you be so kind and push some code change to trigger a new build. After green I will merge. Appreciate it!

**Comment by maicki** [comment]
Build finished successfully although it does not show up. Gonna merge.

**Comment by garrettmoon** [comment]
🙄 https://github.com/TextureGroup/Texture/pull/979/files/e8b8fd651eafbba3836113ca67e48e00b5a932d2#r197000214

**Pull request merged**